Makefile

```diff
@@ -1,5 +1,16 @@
-test:
-	npm test
+PKG:=yarn
 
-lint:
-	npm run lint
+help:
+	@echo
+	@echo "✍🏽  Please use 'make <target>' where <target> is one of the commands below:"
+	@echo
+	@fgrep -h "##" $(MAKEFILE_LIST) | fgrep -v fgrep | sed -e "s/\\$$//" | sed -e "s/##//"
+	@echo
+
+lint: ## lints javascript lines
+	$(PKG) lint
+
+prepare: lint test ## checks if a commit can be created
```

Comment:
Nice target, I suggest just to remove this comment, it not necessary needs to be associated with commit, it's a target that just runs lint and tests like a shortcut for this targets.

Reply:
yep you're right